controllers/AcquisitionSessionController.php

```diff
@@ -0,0 +1,291 @@
+<?php
+//******************************************************************************
+//                                       AcquisitionSessionController.php
+// SILEX-PHIS
+// Copyright © INRA 2018
+// Creation date: 07 Sept, 2018
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\controllers;
+
+use Yii;
+use yii\web\Controller;
+use yii\filters\VerbFilter;
+use app\models\yiiModels\YiiDocumentModel;
+use app\models\wsModels\WSConstants;
+use app\models\wsModels\WSAcquisitionSession;
+use app\components\helpers\SiteMessages;
+
+require_once '../config/config.php';
+
+/**
+ * CRUD actions for Acquisition Session Metadata File
+ * @see yii\web\Controller
+ * @author Arnaud Charleroy <arnaud.charleroy@inra.fr>
+ */
+class AcquisitionSessionController extends Controller {
+    
+    /**
+     * The name of the worksheet which will be modified
+     */
+    CONST PHIS_SHEET_NAME = "HiddenPhis";
+    
+    /**
+     * The type of the document file  
+     * @var string    
+     */
+    private $documentFileType;
```

Comment:
Why do you need to add attributs in the controller ?

Reply:
Done